pytensor/link/numba/dispatch/basic.py

```diff
@@ -61,6 +61,13 @@ def global_numba_func(func):
 def numba_njit(*args, **kwargs):
     kwargs.setdefault("cache", config.numba__cache)
 
+    # Supress caching warnings
```

Comment:
Maybe we should restore the warnings after compilation so that this doesn't impact unrelated code? (https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings)
I guess in that case we actually need the same thing in nutpie...

Reply:
I tried that but it required a wrapper around the function returned by this function, as it is triggered in at least two places: evaluating the function on VM and when dispatching on Composite. I was also worried of the overhead of entering the context and defining the filter every time we call the function on the VM

Reply:
Ah, I see...
In that case, can we add `numba_funcified_fgraph` to the pattern? In that case it still should only apply to our own function.

Reply:
Yes we can do that